SimPEG/directives/directives.py

```diff
@@ -2178,66 +2178,249 @@ def JtJv(v):
 
 
 class UpdateSensitivityWeights(InversionDirective):
+    r"""
+    Sensitivity weighting for linear and non-linear problems.
+
+    This directive computes the root-mean squared sensitivities for the
+    forward simulation(s) attached to the inverse problem, then truncates
+    and scales the result to create cell weights
+    which are applied in the regularization.
+
+    This directive **requires** that the map for the regularization function is either
+    class:`SimPEG.maps.Wires` or class:`SimPEG.maps.Identity`. In other words, the
+    sensitivity weighting cannot be applied for parametric inversion. In addition,
+    the simulation(s) connected to the inverse problem **must** have a ``getJ`` or
+    ``getJtJdiag`` method.
+
+    Parameters
+    ----------
+    every_iteration : bool
+        When ``True``, update sensitivity weighting at every model update; non-linear problems.
+        When ``False``, create sensitivity weights for starting model only; linear problems.
+    threshold : float
+        Threshold value for smallest weighting value
+    threshold_method : {'global', 'percentile', 'amplitude'}
```

Comment:
'amplitude' the default should be listed first

Reply:
Done